go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/llmos-ai/llmos-operator
 
-go 1.22.7
+go 1.23
```

Comment:
This will require changes in multiple places, refer to
https://github.com/search?q=repo%3Allmos-ai%2Fllmos-operator%201.22&type=code